{"uid":"3c006ff397df6190","name":"Testing Scenario 1","fullName":"Scenario1 .Testing Scenario 1","historyId":"8669a77369ca04d22073f5c29c50d880","time":{"start":1689779752226,"stop":1689779794645,"duration":42419},"description":"","descriptionHtml":"","status":"broken","statusMessage":"`cy.visit()` failed trying to load:\n                \n                http://trustoo.nl:3000/\n                \n                We attempted to make an http request to this URL but the request failed without a response.\n                \n                We received this error at the network level:\n                \n                  > Error: connect ETIMEDOUT 13.225.78.20:3000\n                \n                Common situations why this would fail:\n                  - you don't have internet access\n                  - you forgot to run / boot your web server\n                  - your web server isn't accessible\n                  - you have weird network configuration settings on your computer\n                \n                Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Scenario1 `","statusTrace":"CypressError: `cy.visit()` failed trying to load:\n                \n                http://trustoo.nl:3000/\n                \n                We attempted to make an http request to this URL but the request failed without a response.\n                \n                We received this error at the network level:\n                \n                  > Error: connect ETIMEDOUT 13.225.78.20:3000\n                \n                Common situations why this would fail:\n                  - you don't have internet access\n                  - you forgot to run / boot your web server\n                  - your web server isn't accessible\n                  - you have weird network configuration settings on your computer\n                \n                Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Scenario1 `\n                    at http://localhost:59004/__cypress/runner/cypress_runner.js:157807:23\n                    at visitFailedByErr (http://localhost:59004/__cypress/runner/cypress_runner.js:157162:12)\n                    at http://localhost:59004/__cypress/runner/cypress_runner.js:157806:11\n                    at tryCatcher (http://localhost:59004/__cypress/runner/cypress_runner.js:10325:23)\n                    at Promise._settlePromiseFromHandler (http://localhost:59004/__cypress/runner/cypress_runner.js:8260:31)\n                    at Promise._settlePromise (http://localhost:59004/__cypress/runner/cypress_runner.js:8317:18)\n                    at Promise._settlePromise0 (http://localhost:59004/__cypress/runner/cypress_runner.js:8362:10)\n                    at Promise._settlePromises (http://localhost:59004/__cypress/runner/cypress_runner.js:8438:18)\n                    at _drainQueueStep (http://localhost:59004/__cypress/runner/cypress_runner.js:5032:12)\n                    at _drainQueue (http://localhost:59004/__cypress/runner/cypress_runner.js:5025:9)\n                    at ../../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://localhost:59004/__cypress/runner/cypress_runner.js:5041:5)\n                    at Async.drainQueues (http://localhost:59004/__cypress/runner/cypress_runner.js:4911:14)\n                \n                From Node.js Internals:\n                  Error: connect ETIMEDOUT 13.225.78.20:3000\n                      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16)\n                  ","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"suite","value":"Scenario1 "},{"name":"testClass","value":"Scenario1 "},{"name":"testMethod","value":"Testing Scenario 1"},{"name":"package","value":"Scenario1 "},{"name":"resultFormat","value":"allure1"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"874416879076ed87","status":"broken","statusDetails":"`cy.visit()` failed trying to load:\n                \n                http://trustoo.nl:3000/\n                \n                We attempted to make an http request to this URL but the request failed without a response.\n                \n                We received this error at the network level:\n                \n                  > Error: connect ETIMEDOUT 13.225.78.66:3000\n                \n                Common situations why this would fail:\n                  - you don't have internet access\n                  - you forgot to run / boot your web server\n                  - your web server isn't accessible\n                  - you have weird network configuration settings on your computer\n                \n                Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Scenario1 `","time":{"start":1689779269117,"stop":1689779311556,"duration":42439}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"3c006ff397df6190.json","parameterValues":[]}